packages/DataGrid/src/DataGrid.js

```diff
@@ -320,12 +311,18 @@ const DataGrid = React.forwardRef((props, ref) => {
   function handleFocusGrid() {
     const $isBlurred = refContainer.current.querySelector(".grid--is-blurred");
     if ($isBlurred) $isBlurred.classList.remove("grid--is-blurred");
-
-    if (gridShouldHaveFocus) {
-      setGridShouldHaveFocus(false);
-    }
   }
 
+  const itemData = createItemData(
```

Comment:
Creates a memonize `itemData` prop which will be passdown to each of the react-window grids once the React.memo row needs to be consumed `DataGrid.rows.js`